notebooks/evaluation/baseline_evaluation.ipynb

```diff
@@ -0,0 +1,183 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "df = pd.read_csv(\"./../../datasets/baseline.csv\")\n",
+    "\n",
+    "df_train_val, df_test = train_test_split(df, test_size=0.2, random_state=42)\n",
+    "\n",
+    "df_test = df_test.reset_index(drop=True)\n",
+    "\n",
+    "display(df_test)"
```

Comment:
Same here, do not print the whole dataframe

Reply:
output cleared